src/util.rs

```diff
@@ -230,49 +230,67 @@ pub fn print_blocks(
         }
 
         // Get the final JSON from all the widgets for this block
-        let rendered_widgets = widgets
+        let mut rendered_widgets = widgets
             .iter()
             .map(|widget| {
+                let mut w_json: serde_json::Value = widget.get_rendered().to_owned();
                 if alternator {
                     // Apply tint for all widgets of every second block
-                    let mut w_json: serde_json::Value = widget.get_rendered().to_owned();
                     *w_json.get_mut("background").unwrap() = json!(add_colors(
-                        &w_json["background"].to_string()[1..],
-                        &config.theme.alternating_tint_bg
+                        w_json["background"].as_str(),
```

Comment:
Does this still work without trimming the leading "#" from the colour string?

Reply:
Curiously, yes as far as I can see. It's later trimmed again when it's split into RGB

Reply:
Just checked the actual color values in the output and they are correct without this trim. I don't know why this worked before, probably the reference didn't have it trimmed?

Reply:
Weird, when I did the alternating tint PR I remember having to trim it otherwise it would panic

Reply:
Well it works very reliably for me now